packages/Input/src/input.scss

```diff
@@ -87,40 +121,37 @@
   appearance: none;
 }
 
-.ids-input:hover {
-  border-color: var(--ids-input-border-color-hover);
-}
-
 .ids-input--error {
   border-color: var(--ids-input-border-color-error);
 }
 
 .ids-input:focus,
-.ids-input.focus {
+.ids-input.focus,
+.ids-input:active,
+.ids-input.active {
   outline: 0;
   outline-offset: 0;
-  box-shadow:
-    0 0 0 2px var(--ids-input-box-shadow-inner-color-focus),
-    0 0 0 3.5px var(--ids-input-box-shadow-outer-color-focus);
+}
+
+.ids-input:focus,
+.ids-input.focus {
+  box-shadow: var(--focus);
 }
 
 .ids-input:active,
 .ids-input.active {
   border: 1px solid var(--ids-input-border-color-active);
-  border-radius: var(--ids-input-border-radius-active);
-  outline: 0;
-  outline-offset: 0;
   box-shadow: none;
 }
 
 .ids-input:disabled,
-.ids-input--readonly {
-  color: var(--ids-input-color-readonly);
-  background-color: var(--ids-input-background-readonly);
-  border-color: var(--ids-input-border-color-readonly);
+.ids-input--disabled {
+  color: var(--ids-input-color-disabled);
+  background-color: var(--ids-input-background-disabled);
+  border-color: var(--ids-input-border-color-disabled);
   cursor: not-allowed;
 }
 
-.ids-input--readonly {
+.ids-input--disabled {
   cursor: default;
```

Comment:
Juste après avoir configuré le curseur en `not-allowed` vers la ligne 152, il est ensuite remis à `default`.